src/common.build.pre.props

```diff
@@ -110,8 +109,12 @@
         C26813: Use 'bitwise and' to check if a flag is set.
           The MSVC v19.31 toolset has a bug where a pointer to an enum is incorrectly flagged with C26813.
           It's supposed to be fixed with VS 17.2.1 and 17.3.0 and later respectively.
+        C26493: Don't use C-style casts (type.4).
+          The MSVC 14.33.31629 toolset (VS 17.3 Preview 5) has a bug
+          where brace initializations are treated as C-style casts.

```

Comment:
Submitted as https://developercommunity.visualstudio.com/t/C26493-reported-for-brace-initialization/10105646